source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -217,6 +222,20 @@ namespace
         printResults({ "  Total 2D capture time:" }, durations);
     }
 
+    void printCopyDataResults(const std::vector<Duration> durations[], const size_t numCopies)
```

Comment:
`&durations`? Is `[]` necessery?

Reply:
Make `durations` `std::array<std::vector<Duration>>` and pass it by reference `&durations`